app/js/actions.js

```diff
@@ -30,78 +30,78 @@ var AllActions = {
     observe: new Action(            'observe',              'Observe',               0,      7,  1.0, 0.0, 0.0, 'immediate',   1,  'All',           1),
 
     basicSynth: new Action(         'basicSynth',           'Basic Synthesis',      10,      0,  0.9, 0.0, 1.0, 'immediate',   1,  'All',           1),
-    standardSynthesis: new Action(  'standardSynthesis',    'Standard Synthesis',   10,     15,  0.9, 0.0, 1.5, 'immediate',   1,  'All',          31),
-    carefulSynthesis: new Action(   'carefulSynthesis',     'Careful Synthesis',    10,      0,  1.0, 0.0, 0.9, 'immediate',   1,  'Weaver',       15),
-    carefulSynthesis2: new Action(  'carefulSynthesis2',    'Careful Synthesis II', 10,      0,  1.0, 0.0, 1.2, 'immediate',   1,  'Weaver',       50),
-    rapidSynthesis: new Action(     'rapidSynthesis',       'Rapid Synthesis',      10,      0,  0.5, 0.0, 2.5, 'immediate',   1,  'Armorer',      15),
-    flawlessSynthesis: new Action(  'flawlessSynthesis',    'Flawless Synthesis',   10,     15,  0.9, 0.0, 1.0, 'immediate',   1,  'Goldsmith',    37),
-    pieceByPiece: new Action(       'pieceByPiece',         'Piece by Piece',       10,     15,  0.9, 0.0, 1.0, 'immediate',   1,  'Armorer',      50),
+    standardSynthesis: new Action(  'standardSynthesis',    'Basic Synthesis',      10,      0,  1.0, 0.0, 1.2, 'immediate',   1,  'All',          31),
```

Comment:
Is there a reason the name was changed to `Basic Synthesis`?

Reply:
They got rid of levels of ability for this.  You now unlock trait at some level which upgrades your existing abilities.  Riding in vehicle or I'd post the trait name and description of you cared.

Reply:
From a code perspective there should likely be something in the heuristics which prevents it from attempting basic synth when standard is available.  There may already be but once again on the road so can't easily investigate

Reply:
Looks like the same has happened to Rapid Synthesis.

(see https://www.xivcraftingguide.com/51-patch-changes)

Reply:
I added the level to the action tooltip to make it easier to distinguish these actions for now, until a better solution can be determined.